lsp-clients.el

```diff
@@ -702,6 +702,34 @@ finding the executable with `exec-path'."
                   :priority -1
                   :server-id 'iph))
 
+;; Hack
+(defgroup lsp-hack nil
+  "Hack."
+  :group 'lsp-mode
+  :tag "Hack")
+
+(defcustom lsp-clients-hack-command '("hh_client" "lsp" "--from" "emacs")
+  "hh_client command."
+  :group 'lsp-hack
+  :risky t
+  :type 'list)
+
+(defun lsp-hack--ignore (&rest _ignore)
+  "Do nothing and return nil.
+  This function accepts any number of arguments, but ignores them."
+  (interactive)
+  nil)
+
+(lsp-register-client
+ (make-lsp-client :new-connection (lsp-stdio-connection (lambda () lsp-clients-hack-command))
+                  :major-modes '(hack-mode)
+                  :priority -1
+                  :server-id 'hack
+                  ;; ignore some unsupported messages from Nuclide
+                  :notification-handlers (lsp-ht ("telemetry/event" 'lsp-hack--ignore)
```

Comment:
nit: there is a standard emacs function `ignore`.

Reply:
Yes! I actually copied the code and the comment from there. I got confused since it was in the subr package which was not required in this package yet. You are right though that it is a standard library distributed with emacs. I'll update it to use ignore